### PR TITLE
Improving Signal Stability Component.

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@flopezag 

### DESCRIPTION
Defined the size of the temporal window in the facts component in the configuration file
Insert the date & time in the redis together with the value of the data that it is received from the Orion CB.
If the 1st and last value is greater than or equal to the temporal window,we delete the queue and insert the last value on it.
Written Unit Test of the execution of this US

### CODE REVIEW

**Comment by flopezag** [comment]
I do not understand why with this code, but the coverage decrease 5%?

File: facts/myredis.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def insert(self, tenantid, serverid, data):
                               is no list or the number of element is not equal to 4.
         """
         ''' we need to check that data is a list and the exact number of
-        element is equal to 3 - Magic Number
+        element is equal to 4 - Magic Number
         '''
         if isinstance(data, list) and len(data) == 4:
```
Comment (flopezag) [inline facts/myredis.py]: :)

This comment was written in order to fix the Magic Number when we have clear want we want to do here...

Comment (geonexus) [inline facts/myredis.py]: yes, We have everything clear but until it is not implemented, I prefer to preserve the comment

Comment (flopezag) [inline facts/myredis.py]: mmm, I think that we can fix it and then if it is necessary we can change the value in future release. Please apply it.

File: facts/config.py [hunk 2]
```diff
@@ -56,3 +57,7 @@
 # Initialize the content of the config parameters
 for key, value in cfg_defaults.items():
     config.set('common', key, str(value))
+
+windowsize_facts = datetime.timedelta(seconds=10)
+
+fact_attributes = ['serverId', 'CpuValue', 'MemValue', 'DataTime']
```
Comment (flopezag) [inline facts/config.py]: mmm, I like it, imaginative solution that give you detail about the data that you are using.

**Comment by flopezag** [comment]
LGTM

**Pull request merged**